pkg/sql/tests/rsg_test.go

```diff
@@ -333,11 +369,80 @@ func TestRandomSyntaxSchemaChangeColumn(t *testing.T) {
 	})
 }
 
+var ignoredErrorPatterns = []string{
```

Comment:
I'd be a little nervous that people will add new error types without being aware of this? And they won't find out until the nightly runs (and they might not be the ones who notice). But I don't have a better solution.